test/Vagrantfile

```diff
@@ -7,8 +7,8 @@ $JOB_NAME = ENV['JOB_BASE_NAME'] || "LOCAL"
 $K8S_VERSION = ENV['K8S_VERSION'] || "1.10"
 $K8S_NODES = (ENV['K8S_NODES'] || "2").to_i
 $NFS = ENV['NFS']=="1"? true : false
-$SERVER_BOX= "cilium/ubuntu"
-$SERVER_VERSION="77"
+$SERVER_BOX = (ENV['SERVER_BOX'] || "cilium/ubuntu-dev")
```

Comment:
What happens if the versions get out of sync for each box? Is this possible? Shouldn't the version be set based off of the value of `SERVER_BOX`?

Reply:
Same Jenkins job, so should be always in the same version